src/popup/router/components/CheckBox.vue

```diff
@@ -28,6 +25,9 @@ export default {
         this.checkedProxy = val;
       },
     },
+    getType() {
+      return !this.type ? 'checkbox' : this.type;
```

Comment:
Other option would be to set a default value for `type` property: https://vuejs.org/v2/guide/components-props.html#Prop-Validation